src/test/scala/nl.knaw.dans.easy.deposit/docs/DDMSpec.scala

```diff
@@ -537,6 +537,10 @@ class DDMSpec extends TestSupportFixture with DdmBehavior {
           </boundedBy>
         </dcx-gml:spatial>
         <dcterms:license xsi:type="dcterms:URI">http://creativecommons.org/publicdomain/zero/1.0</dcterms:license>
+        <dcterms:language xsi:type="dcterms:ISO639-2">English</dcterms:language>
+        <dcterms:language xsi:type="dcterms:ISO639-2">Dutch</dcterms:language>
```

Comment:
these values are not in iso639-2, should be `nld` or `dut` 
```suggestion
        <dcterms:language xsi:type="dcterms:ISO639-2">nld</dcterms:language>
```

Reply:
@stephanHogenboom I'm not sure your changes are what @lindareijnhoudt intended. You're using `lang.value`, but shouldn't it be `lang.key`? See

https://github.com/DANS-KNAW/easy-deposit-api/blob/d023e0755d2a7dee41e52e587a15555b27236263/src/main/scala/nl.knaw.dans.easy.deposit/docs/DDM.scala#L72

In your changes you changed the example input data, but that isn't how to solve this. Could you test this together with `easy-deposit-ui` as well?